float64 voxel_size
sensor_msgs/PointCloud2 map_cloud
---
vgn/GraspConfig[] grasps
